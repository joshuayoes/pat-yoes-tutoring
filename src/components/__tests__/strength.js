import React from "react"
import renderer from "react-test-renderer"

import Strength from "../strength"

describe("Strength", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Strength           
        svg='../../../static/icon-decades-of-experience.svg'
        alt='Award icon'
        heading='Decades of Experience'
        description='Pat has experience in multiple educational roles in his career, giving him a deep breadth of experience to draw from.'
      />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })

  // it('fails when not required props', () => {
  //   const tree = renderer
  //     .create(<Strength           
  //       alt='Award icon'
  //       heading='Decades of Experience'
  //       description='Pat has experience in multiple educational roles in his career, giving him a deep breadth of experience to draw from.'
  //     />)
  //     .toJSON()
    
  //   expect(tree).toMatchSnapshot()
  // })
})