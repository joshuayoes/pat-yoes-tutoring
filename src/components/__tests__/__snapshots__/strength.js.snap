// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Strength fails when not required props 1`] = `
<div
  className="strength"
>
  <div
    className="img_wrapper"
  >
    <img
      alt="Award icon"
      className="img"
    />
  </div>
  <h2
    className="h2"
  >
    Decades of Experience
  </h2>
  <p
    className="p"
  >
    Pat has experience in multiple educational roles in his career, giving him a deep breadth of experience to draw from.
  </p>
</div>
`;

exports[`Strength renders correctly 1`] = `
<div
  className="strength"
>
  <div
    className="img_wrapper"
  >
    <img
      alt="Award icon"
      className="img"
      src="../../../static/icon-decades-of-experience.svg"
    />
  </div>
  <h2
    className="h2"
  >
    Decades of Experience
  </h2>
  <p
    className="p"
  >
    Pat has experience in multiple educational roles in his career, giving him a deep breadth of experience to draw from.
  </p>
</div>
`;
